lib/matplotlib/figure.py

```diff
@@ -568,6 +568,7 @@ def hold(self, b=None):
             self._hold = b
 
     def figimage(self, X,
+                 resize=False,
```

Comment:
This needs to be the _last_ kwarg in case anyone is using this function with only positional arguments (python 3 has a nice way around that, but alas, we still support 2).